styles/mixins/_atoms.scss

```diff
@@ -29,9 +29,9 @@ $-color-shadow: var.$color-shadow;
 @mixin shadow($active: false) {
   @if $active {
     /* offset-x | offset-y | blur-radius | spread-radius | color */
-    box-shadow: 0px 2px 3px 0px color.scale($-color-shadow, $alpha: -85%);
+    box-shadow: 0px 2px 3px 0px $-color-shadow;
   } @else {
     /* offset-x | offset-y | blur-radius | color */
-    box-shadow: 0px 1px 1px color.scale($-color-shadow, $alpha: -90%);
+    box-shadow: 0px 1px 1px $-color-shadow--active;
```

Comment:
This looks the opposite with the else block (it feels it should this be `--no-active`).

Maybe sticking with `--dark` is better.

Reply:
You're right, this is just...wrong. Will fix.